versions.txt

```diff
@@ -3,6 +3,6 @@
 
 google-cloud-bom:0.230.0:0.231.0-SNAPSHOT
 libraries-bom:26.49.0:26.50.0-SNAPSHOT
-libraries-bom-protobuf3:0.1.0:0.2.0-SNAPSHOT
+libraries-bom-protobuf3:26.49.0:26.50.0-SNAPSHOT
```

Comment:
Can you remove libraries-bom-protobuf3 key in favor of libraries-bom?

Reply:
Updated